src/M6Web/Bundle/LogBridgeBundle/Matcher/Builder.php

```diff
@@ -77,88 +59,35 @@ class Builder implements BuilderInterface
     private $matcher;
 
     /**
+     * @var array
+     */
+    private $filters;
+
+    /**
+     * @var array
+     */
+    private $activeFilters;
+
+    /**
      * __construct
      *
      * @param StatusTypeManager $statusTypeManager Status type manager
-     * @param array             $resources         Resources
      * @param string            $environment       Environment name
+     * @param array             $filters           Filters
+     * @param array             $activeFilters     Active Filters
      */
-    public function __construct(StatusTypeManager $statusTypeManager, array $resources, $environment)
+    public function __construct(StatusTypeManager $statusTypeManager, array $filters, array $activeFilters, $environment)
```

Comment:
Parameters are not in the same order as the phpDoc. Maybe we can add `phpCsFixer` for this kind of check?

Reply:
Yes - But i propose to open an other PR for this change - we can use coke for this one ?